tests/dummy/app/templates/map.hbs

```diff
@@ -18,6 +18,7 @@
     onShow=(action "onErrorMessageShow")
     onHide=(action "onErrorMessageHide")
   }}
+  {{flexberry-error error=error}}
```

Comment:
Error dialog isn't visible on this form, maybe because of some custom styles.
Try to remove map name in map properties, for example, and save form, you'll see dimmer, but you won't see error dialog.